auth/src/main/java/com/firebase/ui/auth/ui/accountlink/WelcomeBackPasswordPrompt.java

```diff
@@ -117,7 +120,7 @@ protected void onCreate(Bundle savedInstanceState) {
     public void onClick(View view) {
         final int id = view.getId();
         if (id == R.id.button_done) {
-            next(mEmail, mPasswordField.getText().toString());
+            run();
```

Comment:
Rather than calling `run` here, make `run()` call out to a no-args method like this:

```java
public void onClick(View view) {
    // ...
    if (id == R.id.button_done) {
        nextWithEmailAndPassword();
    }
    // ...
}

// ...


@Override
public void run() {
    nextWithEmailAndPassword();
}

private voidNextWithEmailAndPassword() {
   next(mEmail, mPasswordField.getText().toString());
}
```